src/config/theme.ts

```diff
@@ -106,16 +133,42 @@ export const baseThemeDef: ThemeOptions = {
   },
   palette: {
     primary: {
-      main: '#1976D2',
+      surface: '#F8F9FB',
```

Comment:
Related to Theron's comment, it looks like Overline text is another place where we are still using the MUI primary color blue

Reply:
Updated to stop using `links` color for overline text in several places, thanks!